### PR TITLE
This directive is to resolve bad UX that occurs on input elements whe…

### PR DESCRIPTION
**Pull request by WesleyPumpkinhead**
…n suggestions are cached. Specifically, this fixes the problem of suggestions persisting on the map template if you didn't select one but did submit a search request by using the return key.

### CODE REVIEW

File: src/core/javascripts/directives/main.js.coffee [hunk 1]
```diff
@@ -224,3 +224,29 @@ angular.module('BB.Directives').directive('bbBackgroundImage', () ->
...
+* @scope true
+*
+* @description
+* Removes focus from an input[type=text] element when return key is pressed
+
+
+* @example
+* <input type='text' blur-on-return></div>
+####
+angular.module('BB.Directives').directive('blurOnReturn', ($timeout) ->
```
Comment (lukeferguson) [inline src/core/javascripts/directives/main.js.coffee]: Can you prefix the directive with bb for consistency please :)

**Comment by lukeferguson** [comment]
@WesleyPumpkinhead  can you please resolve the branch conflicts! :)

**Comment by lukeferguson** [comment]
@WesleyPumpkinhead add a requrie: ngModel clause to the directive to enforce binding with a input element.

**Comment by WesleyPumpkinhead** [comment]
@lukeferguson You can delete this and use pull request #204 instead

**Pull request closed without merging**